packages/fxa-auth-server/lib/senders/emails/templates/cadReminderFirst/index.ts

```diff
@@ -0,0 +1,9 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { cadReminder } from '../../partials';
+
+export const render = () => {
+  return `${cadReminder}`;
+};
```

Comment:
Will we need a file like this for every template? (this review is a first pass at this PR, haven't checked in and ran things). I _think_ you can maybe simplify this entire file to something like `export {default as render} from '../../partials'` if `cadReminder` is the default export, but I didn't test it myself. I can circle back to this later, seems like it could use something though👀

Reply:
After checking into the branch and poking around, I see what this is doing now and what I suggested won't work (even `cadReminder as render`) since this needs to be a function.

We can change L7-9 to `export const render () => cadReminder;` at least.